db/migrate/20230128165117_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
Great job @uisendy 🤝

- [ ] [Optional] You may change the type of `photo` to `string` as the photo should be a link to an image, not a file.